### DISCUSSION TITLE
Racks with Vertical Slots for Devices

### DISCUSSION THREAD

**Discussion by willflo** [netbox-community/netbox#10123] [Ideas]
Hey we do have some racks which have vertical slots for devices. We are currently using those slots for management switches. Could you give us the possiblity to add vertical slots to racks or is there another way to achieve this? Currently all of these devices are unracked in our installation.

Thank you
Best Regards.

**Comment by candlerb** [comment]
Are these racks mounted inside a more traditional rack, or are they completely standalone?

What you can do is create the rack as a "Parent" device with device bays, and install the devices as "Child" devices within those bays. You won't, however, get any graphical rendering of the installed devices, either horizontal or vertical.

**Comment by willflo** [comment]
Those are traditional racks with just some vertial side mounts. Ive added a picture how they are looking. 
Would it be possible to get some kind of editor to also add those mounts to the graphical rendering? 

https://imgur.com/PzvftKa

**Reply by candlerb** [reply]
The SVGs are generated on-the-fly, dynamically in Python code, so are not editable.

I think setting the devices to "unracked" is the best you can do currently, perhaps with a custom field for vertical slot position. You can also upload your own images against a rack, but you'd have to make and maintain those yourself, e.g. in something like drawio.

You *could *use parent/child as I described before, where the parent device is set to unracked. However, you then wouldn't see the devices at all in the rack view (not even in the "unracked" section).  You'd have to click on the parent device, and then click on its Device Bays tab.

**Reply by bitcollector1** [reply]
We have a few of these racks as well and I just made the racks taller and mounted them in the top, so they still show in the rack. Then I made a comment that this is not the correct location.
